wp-content/themes/workingnyc/views/employer-programs/employer-program.twig

```diff
@@ -2,17 +2,17 @@
 {# @For    Post Type #}
 {# @Class  timber-posts/Programs.php #}
 {# For homepage collection #}
-<article class="rounded border border-[#ECEFF2] p-3 w-full h-full">
+<article class="rounded-[5px] border-2 border-[#ECEFF2]  p-3 w-full h-full">
   <header class="pb-2 desktop:pb-3">
-    <img class="w-[140px] h-8 object-contain object-left" loading="lazy" src="{{ post.meta('employer_program_logo').url }}" alt="{{ post.meta('employer_program_logo').alt }}"/>
+    <img class="w-[150px] h-[70px] object-contain object-left p-[4px] pb-0" loading="lazy" src="{{ post.meta('employer_program_logo').url }}" alt="{{ post.meta('employer_program_logo').alt }}"/>
   </header>
   <div>
-    <div class="pb-2 desktop:pb-3">
-      <a href="{{ post.link }}">
-        <h5 class="pb-1 font-[600]">{{ post.employer_program_title }}</h5>
+    <div class="pb-2 desktop:pb-3 max-h-[90px]">
```

Comment:
- [ ] Don't use max heights for anything with content, only min heights, so that the content doesn't get cut off

Reply:
Removed Max-h-[90px]